docs/api.rst

```diff
@@ -0,0 +1,26 @@
+edx_api_doc_tools package
+============================
+
+edx_api_doc_tools.apps module
+-----------------------------
+
+.. automodule:: edx_api_doc_tools.apps
+   :members:
+
+edx_api_doc_tools.conf_utils module
```

Comment:
Can we expose docs for  `edx_api_doc_tools.__init__` instead?

I imagined the `data`/`conf_utils`/`view_utils` separation as being an internal code-organization thing, not an interface.

Reply:
Done.